packages/node/src/lib/metrics-log.ts

```diff
@@ -41,14 +41,16 @@ export interface OutgoingLogBody {
   request: Har;
 }
 
+type LogId = string | string[] | undefined;
```

Comment:
If you wanted, you could include the `uuid` type here and consume it instead of `string`.

Reply:
it was slightly annoying since the `uuid` npm package doesn't play nicely with Node's baked in `UUID` type, but i like this idea — see here! https://github.com/readmeio/metrics-sdks/pull/900/commits/b5c3455790942e4d232f72ee8b60f7a601bb7ef2

Reply:
why are we using an npm package instead of `randomUUID` from `node:crypto`?

Reply:
looks like it only got added in Node v19: https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility

Reply:
doh! thanks for nothing, ide auto-fill

Reply:
(was the `UUID` type added before the uuid function?)

Reply:
wait weird, apparently the Node.js docs say that `randomUUID` has been available since Node 14/15? https://nodejs.org/api/crypto.html#cryptorandomuuidoptions

Reply:
so maybe we could refactor things to use that, at least once we drop support for Node 14